sway/commands/gaps.c

```diff
@@ -0,0 +1,153 @@
+#include <string.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/tree/arrange.h"
+#include "log.h"
+#include "stringop.h"
+
+int read_double(char *str, double *out) {
+	char *end;
+	double v = strtod(str, &end);
+	if (str == end) {
+		*out = 0.0;
+		return 1;
+	}
+	*out = v;
+	return 0;
+}
+
+struct cmd_results *cmd_gaps(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "gaps", EXPECTED_AT_LEAST, 1))) {
```

Comment:
I know this code is basically copied from somewhere else, but I really don't see the point of forcing this assignment to happen inside the conditional.
```c
struct cmd_results *error = checkarg(...);
if (error) {
    ...
}
```
works fine and I would argue looks a lot cleaner.